File: Option.java
// option.java
public class Option {
    private String text;

    public Option() {
        this.text = "";
    }

    public Option(String text) {
        this.text = text;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public void appendText(String additionalText) {
        this.text += " " + additionalText;
    }

    public void clearText() {
        this.text = "";
    }
}

File: Option.cpp
#define _CRT_SECURE_NO_WARNINGS
#include "Option.h"

// Конструктор по умолчанию
Option::Option() : text("") {}

// Конструктор с параметром
Option::Option(const std::string& optionText) : text(optionText) {}

// Метод для получения текста
const std::string& Option::getText() const {
    return text;
}

// Метод для установки текста
void Option::setText(const std::string& optionText) {
    text = optionText;
}
